tests/playwright/vite.config.js

```diff
@@ -0,0 +1,43 @@
+import { join } from 'path';
+import { builtinModules } from 'module';
+import typescript from '@rollup/plugin-typescript';
```

Comment:
why ?

Reply:
So it includes all typescript files in building the final `dist` with vite. Does not work without it.

Reply:
All `.ts` files under tests/playwright? I am not really sure how you do that in other packages, nor what is right, this works. without it it does not work. I do not know how better describe this.

Reply:
ok let's postpone this